assets/src/edit-story/panels/panel/arrow.svg

```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Should be have a global directory for all icons / svgs for more reusability.

Reply:
I feel assets should be located where used unless circumstances require it. So, on first reuse elsewhere it can be moved, but not before.